Python/Product/PythonTools/PythonTools/Intellisense/ProjectAnalyzer.cs

```diff
@@ -2264,9 +2341,19 @@ class ExprWalker : PythonWalker {
                 fileId = analysis.Entry.FileId,
             };
 
-            var resp = await analysis.Entry.Analyzer.SendRequestAsync(req, timeout: timeout).ConfigureAwait(false);
-
-            return resp?.expression;
+            return analysis.Entry.Analyzer.EnsureSingleRequest(
+                typeof(AP.ExpressionForDataTipRequest),
+                req.expr,
+                t => t == req.expr,
```

Comment:
Yeah, it probably should. That's very unlikely to happen in reality (VS only supports one data tip at a time), but I could see it coming up in the test suite.